trestle/core/catalog_interface.py

```diff
@@ -238,6 +238,46 @@ def get_all_groups_from_catalog(self) -> Iterator[cat.Group]:
                 for res in CatalogInterface._get_groups_from_group(my_group):
                     yield res
 
+    def get_statement_label_if_exists(self, control_id: str, statement_id: str) -> Optional[str]:
+        """Get statement label if given."""
+        # TODO: workaround when statement ids contain additional dot at the end
+        # remove this after this is fixed
+        if statement_id[-1] == '.':
+            statement_id = statement_id[0:-1]
```

Comment:
Let's see if we can merge @fsuits' PR first so we can remove this.

Reply:
Will remove after Frank's PR is merged